covid_symptoms/Dockerfile

```diff
@@ -40,17 +37,7 @@ WORKDIR /ctakes
 RUN mvn compile -pl '!ctakes-distribution' -DskipTests
 RUN mvn install -pl '!ctakes-distribution' -DskipTests
 
-WORKDIR /
-RUN mv /ctakes/ctakes-web-rest/target/ctakes-web-rest.war /apache-tomcat-9.0.21/webapps/
-
-RUN rm -rf /root/.m2
-
-ENV TOMCAT_HOME=/apache-tomcat-9.0.21
+FROM tomcat:9.0.70-jre8-temurin
+COPY --from=0 /ctakes/ctakes-web-rest/target/ctakes-web-rest.war $CATALINA_HOME/webapps/
```

Comment:
nit: I like names, rather than `--from=0`, but that's a personal pref and who cares

Reply:
for more than one step, definetely - for a two stage build my first instinct was it wouldn't be a big deal but it is better for readability.